src/components/pages/StudentHome/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import { Layout } from 'antd';
+import 'antd/dist/antd.css';
+import Banner from '../../common/Banner';
+import StudentSideBar from './StudentSidebar';
+import StudentDashboard from './StudentDashboard';
+import '../../../styles/ParentStyles/index.less';
+
+function StudentHome() {
+  const { Content, Sider } = Layout;
+  const [collapsed, setCollapsed] = useState(false);
+
+  const toggleCollapsed = () => {
+    setCollapsed(!collapsed);
+  };
+
+  return (
+    <div className="App">
+      <Layout style={{ width: '100%' }}>
```

Comment:
Here, in your `style` object, is where you'll be able to affect the styles you're seeing when inspecting the active page. (`ant-layout-content`) - try adding `justify-content: inherit` and see if your page doesn't start behaving better here. 😄

Reply:
I was able to override the ant-layout-content CSS in the style sheet by writing this: 

.ant-layout-content {
    display: block !important;
}

It's been resolved on my end.